### PR TITLE
Smashing Magazine - code review

### PR DESCRIPTION
**Pull request by javierbotero**
This is cloning for the Smashing magazine landing page using grey tones for building a heatmap.

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really fantastic job @javierbotero and @Joecleverman!
This is really awesome and you are almost at the end of it all. Please look into the following details thoroughly and request another review :smile: 

#### Three article rows
If you look at the original site, this section is technically missing as you have 4 articles in it instead of 3 as is required. Please look more closely into this.

#### Live link
Please remember to update the live link in your project to point to the actual website. My go to resource for making live links is [Raw Githack](https://raw.githack.com) Please check it out, it's free and easy to use :+1: 

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+      </div>
+
+      <div class="large white-black">
+        <h2>Smashing Newsletter</h2>
+        <img
+          src="https://d33wubrfki0l68.cloudfront.net/ce7741e1659c0366fda51555365194e0bfd136c0/ae578/images/smashing-cat/cat-with-trampet.svg"
+          alt="image"
+        />
+      </div>
+    </div>
```
Comment (mosesogwo) [inline index.html:250]: The horizontal scroll is coming from the styling/sizing for this div (`.form1-container`) or the elements in it. Re-check  your styling for this div to fix the horizontal scroll.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### You have done well @javierbotero , @techbabouche  , however you haven't fixed the **horizontal scroll** issue pointed out by the last reviewer.

I have marked out the section of your html that is causing the scroll. Check the styling and fix the error and you should be good to :airplane: .

**Great job so far. Just a few more steps.**

### Status: Changes Requested!

**Comment by Joecleverman** [comment]
I removed the horizontal scroll bar.

**Review by mosesogwo — APPROVED** [review]
### Great job team.

Your project is approved!

You can now merge to master and submit your project completion form.

Best.

## Status: Approved :airplane:

**Pull request merged**